### DISCUSSION TITLE
Cool demo video

### DISCUSSION THREAD

**Discussion by avin-kavish** [sweepai/sweep#1041] [General]
Hey!

Cool project you've got going here. I saw your demo video and in addition to the functionality, I was impressed by the video editing as well. Just wondering what tool you used to create it? Cheers!

**Comment by Kamamatoshy** [comment]
> Hey!
> 
> Cool project you've got going here. I saw your demo video and in addition to the functionality, I was impressed by the video editing as well. Just wondering what tool you used to create it? Cheers!

Yes I'm interested to because u can't even get in to see the demo video please help guys

**Comment by wwzeng1** [comment]
Thanks! We worked with our friend (a designer) to create it.
